apps/staging_sync/app.py

```diff
@@ -0,0 +1,132 @@
+import os
+import re
+import subprocess
+import sys
+from pathlib import Path
+
+import streamlit as st
+import streamlit.components.v1 as components
+from git.exc import GitCommandError
+from git.repo import Repo
+from rich.ansi import AnsiDecoder
+from rich.console import Console
+from rich.terminal_theme import MONOKAI
+
+from etl.paths import BASE_DIR
+
+CURRENT_DIR = Path(__file__).resolve().parent
+
+
+def main():
+    st.title("etl-staging-sync")
+    st.markdown(
+        """
+    Sync charts and revisions from staging server to production.
```

Comment:
I am not sure I understand how the app operates.

Isn't it designed to get charts from an arbitrary branch (source) and sync them toan arbitrary server (target), i.e. not necessarily production?

If so, I think that talking about "target" and "production" can be confusing. We should probably just use "target server" or something like that. Maybe explain somewhere that usually we will have target=live